portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,12 @@ function addRandomKeyword() {
   const keywordContainer = document.getElementById('keyword-container');
   keywordContainer.innerText = keyword;
 }
+
+/**
+ * Fetches comments from the server and adds it to the DOM.
+ */
+async function getComments() {
```

Comment:
I would expect a function named getX to return X.

This function does not do so.

Would it be possible to rename this function?

Reply:
Yes. I previously named it to "comments", because they are meant to be under the "Comments" section later. 
While you are right! I have changed all descriptions from "comments" to "data".

Reply:
Have changed the name to "fetchData" to make it clear.